wiki/Beatmap/id.md

```diff
@@ -1,12 +1,118 @@
+
+<!-- TODO: after https://github.com/ppy/osu-web/issues/5852 is resolved, this section will need an update -->
+
+Setelah submisi, sebuah beatmap akan mendapatkan tambahan tempat khusus untuk metadata, seperti deskripsi, bahasa, genre, dan penanda konten eksplisit yang dapat diubah oleh pembuat beatmap di situs web. Beatmap juga mendapatkan [judul teks](Title_Text), yang dapat diubah dengan meminta bantuan kepada [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team).
+
+### Identifikasi
+
+Setiap beatmap yang diunggah akan mendapatkan pengenal numerik (`BeatmapSetID`), yang dapat dilacak di situs web dan melalui [osu!api](/wiki/osu!api). Tingkat kesulitan pada sebuah beatmap juga mempunyai pengenal numerik masing-masing (`BeatmapID`). URL yang mengarah ke tab tingkat kesulitan tertentu pada halaman beatmap menyertakan pengenal numerik tambahan dan memiliki format sebagai berikut:
+
+```
+https://osu.ppy.sh/beatmapsets/{BeatmapSetID}#{GameMode}/{BeatmapID}`
+```
+
+### Kategori beatmap
+
+*Artikel utama: [Kategori beatmap](Category)*
+
+Terdapat beberapa kategori pada beatmap yang telah diunggah, yang dapat berubah seiring berjalannya waktu:
+
+- [Graveyard](Category#graveyard)
+- [Work in Progress / Pending](Category#work-in-progress-and-pending)
+- [Qualified](Category#qualified)
+- [Ranked](Category#ranked)
+- [Approved](Category#approved)
+- [Loved](Category#loved)
+
+Beberapa kategori mempunyai [kriteria submisi](/wiki/Ranking_Criteria) dan memungkinkan beatmap memiliki [papan peringkat](#leaderboards) sementara atau permanen. Cara yang paling disukai untuk mendapatkannya adalah melalui [prosedur beatmap ranking](/wiki/Beatmap_ranking_procedure), sedangkan opsi kedua adalah dengan menjadikan beatmap [loved](Category#loved).
+
+## Mengunduh beatmap
+
+Beatmap dapat didapatkan melalui beberapa cara yang berbeda:
+
+### Daftar beatmap
+
+Beatmap dari berbagai kategori tersedia pada [daftar beatmap](https://osu.ppy.sh/beatmapsets) di situs web. Beatmap-beatmap tersebut dapat diurutkan berdasarkan dengan keinginan dan kebutuhan pengguna (mis. beatmap [osu!taiko](/wiki/Game_mode/osu!taiko) yang dibuat oleh mapper tertentu), lalu diunduh satu per satu. [osu!supporter](/wiki/osu!supporter) memiliki beberapa filter tambahan yang tidak dimiliki oleh pengguna biasa, seperti fitur untuk melihat beatmap di mana mereka mencapai peringkat tertentu.
+
+### osu!direct
+
+osu!direct merupakan daftar beatmap yang dibuat di dalam klien gim, yang digunakan sebagai akses cepat sekaligus menyediakan serangkaian fitur yang sama seperti daftar beatmap yang ada di situs web. Fitur ini hanya tersedia bagi pengguna [osu!supporter](/wiki/osu!supporter).
```

Comment:
osu!direct itself is a feature tho, not a "list"

`su!direct merupakan fitur yang dapat digunakan untuk mengakses daftar beatmap dari dalam klien permainan,...`